src/components/PhoneNumberInput/index.js

```diff
@@ -1,53 +1,72 @@
 import { h, Component} from 'preact'
-import PhoneNumber, {isValidPhoneNumber} from 'react-phone-number-input'
+import PhoneInput from 'react-phone-number-input/native'
+import {parsePhoneNumberFromString} from 'libphonenumber-js'
 
-import classNames from 'classnames';
-import {localised} from '../../locales'
-
-
-import 'react-phone-number-input/rrui.css'
+// import 'react-phone-number-input/rrui.css'
```

Comment:
If you don't need an import, can you just remove it completely?

Reply:
oops...forgot to remove it!